pkgs/applications/editors/neovim/tests.nix

```diff
@@ -133,14 +133,15 @@ rec {
     configure.pathogen.pluginNames = [ "vim-nix" ];
   };
 
-  nvimWithLuaPackages = wrapNeovim2 "with-lua-packages" (makeNeovimConfig {
+  nvimWithLuaPackages = wrapNeovim2 "-with-lua-packages" (makeNeovimConfig {
```

Comment:
```suggestion
  nvimWithLuaPackages = wrapNeovim2 "with-lua-packages" (makeNeovimConfig {
```

Reply:
the original commiter asked me to do the change after I had done it myself to keep all tests consistent.